Updated Path to Production link and /go-live contents (#84)
department-of-veterans-affairs/developer-portal: src/content/goLive.mdx

# Path to Production for API Clients

## Background
The Lighthouse program is moving VA towards an Application Programming Interface (API) first driven digital enterprise, which will establish the next generation open management platform for Veterans and accelerate transformation in VA's core functions, such as Health, Benefits, Burial and Memorials. This platform will be a system for designing, developing, publishing, operating, monitoring, analyzing, iterating and optimizing VA’s API ecosystem. 

## Overview 

The VA API uses token-based authentication. Clients must provide a token with each HTTPS request. This token is issued by the VA. 

## Development API Access

> To receive a developer token to access an API in a test environment, go to <https://developer.va.gov/apply>.  If you are having trouble using <https://developer.va.gov/apply> or have questions, email [api@va.gov](mailto:api@va.gov) and we will get back to you quickly!

Reviewer: ```suggestion
To receive a developer token to access an API in a test environment, [submit an application](/apply).  If you are having trouble with the form or have questions, email [api@va.gov](mailto:api@va.gov) and we will get back to you quickly!
```

Author: @kalilsn got with Alex about this. His suggestion: "To receive a developer token to access an API in a test environment, [submit a request](/apply).  If you are having trouble or have questions, email [api@va.gov](mailto:api@va.gov) and we will get back to you quickly!"

Would you be OK with that instead?

Next, the following criteria needs to be met and verified by the API team. 

- You are a US based company

> - You agree to the [Terms of Service](https://developer.va.gov/explore/terms-of-service)

Reviewer: ```suggestion
- You agree to the [Terms of Service](/explore/terms-of-service)
```
